src/main/java/frc/team166/robot/subsystems/LED.java

```diff
@@ -0,0 +1,333 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.team166.robot.subsystems;
+
+import javax.lang.model.util.ElementScanner6;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.Victor;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.team166.chopshoplib.commands.SubsystemCommand;
+
+public class LED extends Subsystem {
+    //these will be changed from Victors to something else when we get real hardware...
+    Victor blue = new Victor(1);
+    Victor green = new Victor(2);
+    Victor red = new Victor(3);
```

Comment:
Do we know what these will be yet?

Reply:
They will need to be DigitalOutputs... Along with a bunch of other changes to make that work...